handlers/_swagger.js

```diff
@@ -259,48 +260,46 @@ function registerRoute(app, auth, additionalMiddleware, method, path, data, allo
                     var isJson = typeof body === 'string'; //body can come in as JSON or object, we want object
                     body = isJson ? JSON.parse(body) : body;
```

Comment:
there is more safety we should build in around this ... see the comment on the send below